cpp/src/io/utilities/row_selection.hpp

```diff
@@ -27,14 +27,14 @@ namespace cudf::io::detail {
  * @brief Adjusts the input skip_rows and num_rows options to the actual number of rows to
  * skip/read, based on the number of rows in the ORC file(s).
  *
- * @param skip_rows_opt skip_rows as passed by the user
- * @param num_rows_opt num_rows as passed by the user
+ * @param skip_rows skip_rows as passed by the user
+ * @param num_rows_opt Optional num_rows as passed by the user
  * @param num_source_rows number of rows in the ORC file(s)
  * @return A std::pair containing the number of rows to skip and the number of rows to read
  *
  * @throw std::overflow_exception The requested number of rows exceeds the column size limit
  */
 std::pair<uint64_t, size_type> skip_rows_num_rows_from_options(
-  uint64_t skip_rows_opt, std::optional<size_type> const& num_rows_opt, uint64_t num_source_rows);
+  uint64_t skip_rows, std::optional<size_type> const& num_rows_opt, uint64_t num_source_rows);
```

Comment:
Rename to make clear that this is not an optional.

Reply:
OMG, took me a minute :)
`_opt` is not for optional, it stands for option :D 
Maybe both `_opt` suffixes can be removed, they don't seem to work as intended.

Reply:
Done.